docs/integrations/amazon-aws/aurora-mysql-ulm.md

```diff
@@ -229,7 +229,7 @@ To collect AWS CloudTrail events, do the following:
 1. Configure a [Hosted Collector.](/docs/send-data/hosted-collectors/configure-hosted-collector)
 2. Add an [AWS CloudTrail Source](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source.md) to the Hosted Collector, providing the following information:
    * **Name** - Enter a name to display for the new Source.
-   * **Description** - Enter an optional description.
+   * **Description** - (Optional) Enter a description of the Source.
```

Comment:
In my recent PR, you'd indicated formatting should be like this, with a period instead of dash

```
**Source Host**. (Optional) Enter any string to...
```

Reply:
Nope. No hypens. Periods. It's been that way in the style guide for 5 years.